docs/source/index.rst

```diff
@@ -5,10 +5,65 @@ Autosubmit API
 ##############
 
 
+.. raw:: html
+
+   <div>
+      <div class="row gap-4 mb-4 wrap gap-1">
+         <div class="col" style="min-width: min(400px, 100%)"> 
+         The <b>Autosubmit API</b> is an open-source Python web application that aims to monitor, analyze, and control workflows created and managed with the Autosubmit workflow manager
+         </div>
+         <div class="col d-flex flex-column justify-content-center p-1 bg-black text-white shadow" style="min-width: min(400px, 100%)">
+               <div class="px-2 py-1 font-monospace">$ pip install autosubmit_api</div>
+               <div class="px-2 py-1 font-monospace">$ autosubmit_api start</div>
+         </div>
+      </div>
+   </div>
+
+
 .. toctree::
    :maxdepth: 2
 
    /installation/index
    /configuration/index
    /usage/index
    /guides/index
+
+
+
+.. raw:: html
+
+   <div class="mt-4">
+      <h4 class="text-center mb-4">More about Autosubmit ecosystem</h4>
+      <div class="row gap-4 mb-4 wrap gap-1">
+         <div class="col pt-3 border rounded" style="min-width: 20rem;">
+             <a class="fs-4 fw-semibold" href="https://autosubmit.readthedocs.io/" target="_blank" rel="noreferrer" rel="noopener">
+               Autosubmit workflow manager
+             </a>
+             <p class="mt-2">An open source Python experiment and workflow manager used to manage complex workflows on Cloud and HPC platforms</p>
+         </div>
+         <div class="col pt-3 border rounded" style="min-width: 20rem;">
+             <a class="fs-4 fw-semibold" href="https://autosubmit-gui.readthedocs.io/" target="_blank" rel="noreferrer" rel="noopener">
+               Autosubmit GUI
+             </a>
+             <p class="mt-2">Graphical User Interface that aims to provide users with complex information from the workflow execution of scientific experiments managed by Autosubmit</p>
```

Comment:
Maybe in the future we should review and say something like "simplified information" :laughing: I mean, the GUI shows the information/data that's complex, but it makes it simpler :slightly_smiling_face:

Reply:
That makes a lot of sense. Changing it!